openedx_lti_tool_plugin/deep_linking/tests/test_views.py

```diff
@@ -0,0 +1,277 @@
+from openedx_lti_tool_plugin.deep_linking.tests import MODULE_PATH
+from openedx_lti_tool_plugin.deep_linking.views import DeepLinkingFormView, DeepLinkingView
+
+MODULE_PATH = f'{MODULE_PATH}.views'
+
+
+@patch.object(DeepLinkingView, 'tool_config', new_callable=PropertyMock)
+@patch.object(DeepLinkingView, 'tool_storage', new_callable=PropertyMock)
+@patch(f'{MODULE_PATH}.DjangoMessageLaunch')
+@patch(f'{MODULE_PATH}.validate_message')
+@patch(f'{MODULE_PATH}.redirect')
+class TestDeepLinkingViewPost(TestCase):
+    """Test ResourceLinkLaunchView post method."""
+
+    def setUp(self):
+        """Set up test fixtures."""
+        super().setUp()
+        self.view_class = DeepLinkingView
+        self.factory = RequestFactory()
+        self.url = reverse('1.3:deep-linking:root')
+        self.request = self.factory.post(self.url)
+
+    def test_with_deep_linking_request(
+        self,
+        redirect_mock: MagicMock,
+        validate_message_mock: MagicMock,
+        message_launch_mock: MagicMock,
+        tool_storage_mock: MagicMock,
+        tool_conf_mock: MagicMock,
+    ):
+        """Test with LtiResourceLinkRequest."""
+        self.assertEqual(
+            self.view_class.as_view()(self.request),
+            redirect_mock.return_value,
+        )
+        message_launch_mock.assert_called_once_with(
+            self.request,
+            tool_conf_mock(),
+            launch_data_storage=tool_storage_mock(),
+        )
+        validate_message_mock.assert_called_once_with(message_launch_mock())
+        message_launch_mock().get_launch_id.assert_called_once_with()
+        message_launch_mock().get_launch_id().replace.assert_called_once_with('lti1p3-launch-', '')
+        redirect_mock.assert_called_once_with(
+            f'{app_config.name}:1.3:deep-linking:form',
+            launch_id=message_launch_mock().get_launch_id().replace(),
+        )
+
+    @patch.object(DeepLinkingView, 'http_response_error')
+    def test_with_lti_exception(
+        self,
+        http_response_error_mock: MagicMock,
+        redirect_mock: MagicMock,
+        validate_message_mock: MagicMock,
+        message_launch_mock: MagicMock,
+        tool_storage_mock: MagicMock,
+        tool_conf_mock: MagicMock,
+    ):
+        """Test with LtiException."""
+        exception = LtiException('Error message')
+        message_launch_mock.side_effect = exception
+
+        self.assertEqual(
+            self.view_class.as_view()(self.request),
+            http_response_error_mock.return_value,
+        )
+        message_launch_mock.assert_called_once_with(
+            self.request,
+            tool_conf_mock(),
+            launch_data_storage=tool_storage_mock(),
+        )
+        validate_message_mock.assert_not_called()
+        redirect_mock.assert_not_called()
+        http_response_error_mock.assert_called_once_with(exception)
+
+    @patch.object(DeepLinkingView, 'http_response_error')
+    def test_with_deep_linking_exception(
```

Comment:
Same as last comment about name change.
```suggestion
    def test_with_deep_linking_exception(
```